README.md

```diff
@@ -13,26 +13,29 @@ In order to set your environment up to run the code here, first install all requ
 pip install -r requirements.txt
 ```
 
+Rename example.env to .env and edit the variables appropriately.
+MODEL_TYPE supports LlamaCpp or GPT4All
+
 Then, download the 2 models and place them in a folder called `./models`:
```

Comment:
There is no need to place them in ./models anymore, maybe we should express this differently

Reply:
Better wording now?